tests/integration/security/authorization_test.go

```diff
@@ -684,6 +691,7 @@ func TestAuthorization_EgressGateway(t *testing.T) {
 // TestAuthorization_TCP tests the authorization policy on workloads using the raw TCP protocol.
 func TestAuthorization_TCP(t *testing.T) {
 	framework.NewTest(t).
+		Features("security.authorization.tcp-deny").
```

Comment:
why is this `tcp-deny`? I thought this also cover tcp allow cases.

Reply:
change it to tcp